server/models/User.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+  username: { type: String },
+  email: { type: String },
```

Comment:
We will need to add validation at the Mongoose layer in a future PR

Reply:
Okay I added some validation to the user model and just required the content of the messages model.